haystack_argilla/base.py

```diff
@@ -21,30 +28,89 @@ def __init__(
             api_key: Optional[str] = None,
             ) -> None:
         
+        # Add event handlers to Agent
         agent.callback_manager.on_agent_start += self.on_agent_start
         agent.callback_manager.on_agent_final_answer += self.on_agent_final_answer
         agent.callback_manager.on_agent_finish += self.on_agent_finish
 
-        # Import Argilla modules
+        # Import Argilla 
         try:
```

Comment:
can you create a `requirements.txt`? I assume that we will add `argilla` and `haystack` so we can avoid the import error check.